### ISSUE TITLE
Can't build the docker image.

### ISSUE THREAD

**Issue by cryptcoin-junkey** [ElementsProject/lightning#2941]
### Issue and Steps to Reproduce

`python3-mako` was added by #2829. But there is no apk named `python3-mako` in the official repository.

```
git clone
docker build .
```

```
Sending build context to Docker daemon  45.08MB
Step 1/35 : FROM alpine:3.7 as builder
 ---> 6d1ef012b567
Step 2/35 : RUN apk add --no-cache      ca-certificates      autoconf      automake      build-base      libressl      libtool      gmp-dev      python      python-dev      python3      python3-mako      sqlite-dev      wget      git      file      gnupg      swig      zlib-dev
 ---> Running in 157fa933825f
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  python3-mako (missing):
    required by: world[python3-mako]
The command '/bin/sh -c apk add --no-cache      ca-certificates      autoconf      automake      build-base      libressl      libtool      gmp-dev      python      python-dev      python3      python3-mako      sqlite-dev      wget      git      file      gnupg      swig      zlib-dev' returned a non-zero code: 1
```

**Comment by cryptcoin-junkey** [comment]
It should be `py3-mako`. see also: https://pkgs.alpinelinux.org/packages?name=py3-mako&branch=v3.7

**Comment by NicolasDorier** [comment]
@cryptcoin-junkey yeah we need to update the packages I found out in https://github.com/ElementsProject/lightning/pull/2907

**Comment by cdecker** [comment]
Fixed by #2942
